algorithms/src/Kokkos_Sort.hpp

```diff
@@ -566,9 +568,14 @@ struct min_max_functor {
 
 }  // namespace Impl
 
-template <class ExecutionSpace, class ViewType>
-std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
-    const ExecutionSpace& exec, ViewType const& view) {
+template <class ExecutionSpace, class DataType, class... Properties>
+std::enable_if_t<(Kokkos::is_execution_space<ExecutionSpace>::value) &&
+                 (!SpaceAccessibility<
+                     HostSpace, typename Kokkos::View<DataType, Properties...>::
+                                    memory_space>::accessible)>
```

Comment:
Why do you put parentheses around boolean values?

Reply:
I can easily see where one condition ends. Can be removed though if that brokes any Kokkos's coding style.

Reply:
I don't like it but we don't have a rule for this